src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -246,14 +245,29 @@ def _gen_export_id(self):
                 break
         return nid
 
+    def _check_rados_notify(self, ioctx, obj):
+        sleep_time = 0
+        while sleep_time < 60:
+            try:
+                ioctx.notify(obj)
+                break
+            except Exception as e:
+                if e.errno == e.ETIMEDOUT:
+                    sleep_time += 10
+                    log.exception(f"Ganesha timed out {e}")
+                    time.sleep(sleep_time)
+                else:
+                    raise
+
+
     def _write_raw_config(self, conf_block, obj, append=False):
         raw_config = GaneshaConfParser.write_conf(conf_block)
         with self.mgr.rados.open_ioctx(self.rados_pool) as ioctx:
             if self.rados_namespace:
                 ioctx.set_namespace(self.rados_namespace)
             if append:
                 ioctx.append(obj, raw_config.encode('utf-8'))
-                ioctx.notify(obj)
+                self._check_rados_notify(ioctx, obj)
```

Comment:
is 

https://github.com/ceph/ceph/blob/07358ef9045efa65061a0ebb354c1f4093f09d8e/src/pybind/mgr/cephadm/services/nfs.py#L160

also affected by this?

Reply:
No, the object gets created successfully. Since ganesha is failing, I hope this will help to check ganesha logs: https://github.com/varshar16/ceph/commit/e016f5a0640e7e60e1b6eb6461b54e8ea6c3bc5e